mon-pix/app/components/user-certifications-detail-header.js

```diff
@@ -18,6 +20,18 @@ export default class UserCertificationsDetailHeader extends Component {
     return parseISODateOnly(this.args.certification.birthdate);
   }
 
+  get isUserFrenchReader() {
+    return this.currentUser.user && this.currentUser.user.lang === 'fr';
+  }
+
+  get displayCertificationResultsExplanation() {
+    return this.args.certification.version === 3 && (this.currentDomain.isFranceDomain || this.isUserFrenchReader);
```

Comment:
ça vaudrait pas le coup d'avoir un getter pour vérifier cette condition direct dans le model `this.args.certification.version === 3` ?